src/10_functions.py

```diff
@@ -2,6 +2,12 @@
 
 # YOUR CODE HERE
 
+
+def is_even(num):
+    if num % 2 == 0:
+        return True
+
```

Comment:
Good job here. What if the value of num was odd though? What would this function return in that case? How could we change this to return `False` for all odd numbers?